app/javascript/components/generic-objects-form/generic-objects-form.schema.js

```diff
@@ -198,7 +195,7 @@ const createSchema = (initialValues = {}, edit, promise) => {
           label: __('Custom Image File'),
           name: 'file_upload',
           type: 'file',
-          validate: [{ type: 'file', maxSize: 5000000 }],
+          // validate: [{ type: 'file', maxSize: 5000000 }],
```

Comment:
We can remove them if not needed

Reply:
These are needed I just forgot to uncomment them. They were breaking the form but should be fine now since pr: #8333 got merged